web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -181,10 +181,16 @@
             if (config.context) {
               mapReady = gnOwsContextService.loadContextFromUrl(
                 config.context, map);
+            } else {
+              //Failback needed for controllers that rely on map creation
```

Comment:
Very dummy promise, but there are other parts of the code waiting for a promise here